fontforge/sfd.c

```diff
@@ -3326,7 +3326,9 @@ int SFDWriteBak(char *filename,SplineFont *sf,EncMap *map,EncMap *normal) {
     if ( ret && sf->compression!=0 ) {
 	unlink(buf2);
 	buf = malloc(strlen(filename)+40);
```

Comment:
Probably best to switch this to:

```
    qbuf = g_shell_quote(filename);
    buf = malloc(strlen(qbuf)+40);
```

given that `40` is presumably sized against `compressors` and `g_shell_quote` doesn't document how much it may extend the string length.